pkg/virt-api/rest/subresource_test.go

```diff
@@ -1365,23 +1427,34 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 				vm.Status.MemoryDumpRequest = existingMemDumpReq
 			}
 
-			patch, err := generateVMMemoryDumpRequestPatch(vm, memDumpReq, removeReq)
+			resPatch, err := generateVMMemoryDumpRequestPatch(vm, memDumpReq, removeReq)
 			if expectError {
 				Expect(err).To(HaveOccurred())
 			} else {
 				Expect(err).ToNot(HaveOccurred())
 			}
 
-			fmt.Println(patch)
-			Expect(patch).To(Equal(expectedPatch))
+			patches := patch.New()
+			switch {
+			// Add request
+			case expectedMemoryDumpReqTest == nil && expectedMemoryDumpReq != nil:
+				patches.Test("/status/memoryDumpRequest", expectedMemoryDumpReqTest)
+				patches.Add("/status/memoryDumpRequest", expectedMemoryDumpReq)
```

Comment:
Makes sense to add `patches.TestAndAdd` since we also have `patches.TestAndReplace`?

Reply:
Not sure. We should also then add `AddOrTestAndReplace`. It because a bit lengthy imo

Reply:
`patches.TestAndAdd` sounds like a low hanging fruit to me.